CryptoAnalysis/src/main/java/crypto/analysis/CrySLRulesetSelector.java

```diff
@@ -101,4 +117,32 @@ public static CrySLRule makeSingleRule(String rulesBasePath, RuleFormat ruleForm
 		}
 		return rules;
 	}
+	
+	public static List<CrySLRule> makeFromZip(File resource) {
+		List<CrySLRule> rules = Lists.newArrayList();	
+
+		if(resource.exists()) {
+			String resourceFileExtension = Files.getFileExtension(resource.getAbsolutePath());
+			boolean acceptedFormat = false;
+			
+			for(ZipFormat zipFormat : ZipFormat.values()) {
+				if(resourceFileExtension.equals(zipFormat.getFileExtension())) {
+					acceptedFormat = true;
+				}
+			}
+			if(!acceptedFormat) {
+				System.err.println("The format is not supported yet: " + resourceFileExtension);
+				return rules;
+			}
+		} else {
+			System.err.println("The specified path is not a file: " + resource);
+			return rules;
+		}
+	
+		rules = CrySLRuleReader.readFromZipFile(resource);	
+		if (rules.isEmpty()) {
+			System.out.println("No CrySL rules found in " + resource);
```

Comment:
s.o. although i'm not sure. this is not really an error but the analysis can not proceed without any rules. What's your opinion on that?

Reply:
I would throw an exception like in the previous cases...If not the user could be confused why the analysis doesn't find any misuse..instead of being aware of the missing rules